### DISCUSSION TITLE
Error to add date

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#12646] [General - Components / Directives / etc]
Hello Friends 

Am having an issue i wanted to add 1 year on a date that I specified but its giving me an Error am unable to solve can someone please help 

Code example :
    let newDate = addToDate(Date.now(), { years: 1 });

Error :
 Error in v-on handler: "TypeError: t["get".concat(...)] is not a function"

I have followed this example in quasar :  [adde](https://quasar.dev/quasar-utils/date-utils#add-subtract)

Thanks

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/MWOZLqo?editors=1011](https://codepen.io/milospaunovic/pen/MWOZLqo?editors=1011)
